src/VersionsCheck.php

```diff
@@ -65,6 +79,29 @@ public function getOutput()
         return implode("\n", $output)."\n";
     }
 
+    /**
+     * Version comparator bridge to handle BC with old composer versions.
+     *
+     * This method is public only for PHP 5.3 BC and SHOULD NOT be used.
+     * Deprecate it and remove it on next major when PHP 5.3 support will be droped.
+     *
+     * @param string $version1
+     * @param string $operator
+     * @param string $version2
+     *
+     * @return bool
+     */
+    public function versionCompare($version1, $operator, $version2)
```

Comment:
I have to make it public to be BC with the ugly PHP 5.3. See method comment. :-)